components/alert/alert.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>alert</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"rel="stylesheet">
+    <link rel="stylesheet" href="/css/common.css">
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="./alert.css">
+</head>
+<style>
+    .alert-container{
+  
+    display: flex;
+    flex-direction: column;
+    margin: 20px;
+    align-items: center;
+    min-height:55vh;
+    justify-content: space-around;
+}
+
+</style>
```

Comment:
keep all the CSS in a separate file.

Reply:
as of now I am just building components, and not the documentation page. so I added basic styling for components in that page itself.
I have added these CSS intentionally, I will delete it after some time

Reply:
yes made some changes can you review once again